### DISCUSSION TITLE
go 1.17不能按readme顺畅安装，老是有报错，这里是不是要做下兼容

### DISCUSSION THREAD

**Discussion by WeiJiadong** [go-kratos/kratos#1418] [Ideas]
<img width="698" alt="e7c2b83e249319907882e71900b3e4b" src="https://user-images.githubusercontent.com/10074838/131991491-f59ffef3-e09b-4466-85b1-78988f2be247.png">
<img width="499" alt="225dd58a9eb74a977a70c4ad496adb1" src="https://user-images.githubusercontent.com/10074838/131991508-62f8e16c-9e47-4a88-be23-875505ea259d.png">
<img width="799" alt="ab94f81723f66d893b8f7333b719192" src="https://user-images.githubusercontent.com/10074838/131991537-f371a0bf-21bd-4025-b4e3-8c606e23294d.png">
<img width="551" alt="f0f4803e66c0efda812ec5d1f8fc520" src="https://user-images.githubusercontent.com/10074838/131991553-1471a638-ea69-4709-ad5c-8bdb545cda68.png">

**Comment by overstarry** [comment]
go get prints a deprecation warning when installing commands outside the main module (without the -d flag). go install cmd@version should be used instead to install a command at a specific version, using a suffix like @latest or @v1.2.3. In Go 1.18, the -d flag will always be enabled, and go get will only be used to change dependencies in go.mod.
